master/static/srv/notebook-template.ipynb

```diff
@@ -24,6 +24,25 @@
         "### Environment"
       ]
     },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# JupyterLab Notebooks in Determined",
+        "\n",
+        "     # Determined makes managing the environment and resources of JupyterLab easy by using containerized instances.\n",
+        "     # For more information on Determined, please look at our documentation (https://docs.determined.ai/latest/).\n",
+        "\n",
+        "# Notebook Persistence: "
+        "     # Users should save any relevant notebook outputs to shared_fs directory to ensure their work is persisted.\n",
+        "\n",
+        "# Notebook Idle Timeout: "
+        "     # JupyterLab Notebooks in Determined have a configurable timeout that will automatically shut down the notebook instance\n",
+        "     # after the desired timeout period if no kernels or terminal are running. This can be set via the idle_timeout key in the notebook config.\n",
```

Comment:
Add 
```
Note that if you open a Notebook file it might open a kernel for you. And the kernels and the terminals will not be shut down automatically. You need to manually shut down the kernels to make idle timeout effective. 
```